include/model/GPRModel.hpp

```diff
@@ -41,75 +41,28 @@
 namespace libcozmo {
 namespace model {
 
+/// This model assumes it using SE2-based state vector as its state input and
+/// object oriented-based action vector as its model input.
+/// Addtionally, path to model.pkl passed in as construction argument
+/// needs to be the absolute path.
```

Comment:
You're not passing in a path to load the model anywhere. I assume that's in model framework. If so, you don't need this comment